frontend/src/views/AccountPages/AccountPage/AccountPage.js

```diff
@@ -82,9 +83,28 @@ export default function Page(props) {
             </GridContainer>
             <div className={classes.description}>
               <p>
-                {description}
+                {userInfo.isAdmin
+                  ? "Who cares about your subscription? You're the admin!"
+                  : "Welcome back!"}
               </p>
+
             </div>
+
+            {activeMemberships.length === 0
+              ? "You don't have any active memberships currently"
+              : activeMemberships.map((membership, index) => <div key={index} >
```

Comment:
Does membership have an id? That would be a better key.

Reply:
I'd recommend extracting what's inside the `map` into it's own component. Then you can also get rid of the `let remDays` and bring it inside that component and make it's own fucntion instead of mixing it in with the jsx.

Reply:
Changed so that the key is the id
Made component for what's inside the map